pkg/openapiclient/local_files.go

```diff
@@ -65,7 +65,7 @@ func (k *localFilesClient) Paths() (map[string]openapi.GroupVersion, error) {
 		}
 
 		ext := strings.ToLower(filepath.Ext(f.Name()))
-		if ext != "json" && ext != "yml" && ext != "yaml" {
```

Comment:
@alexzielenski I would somewhat expect this code to leave in openapi. Or wait, don't we already have that code in k8s? We should move the code from k8s into kube-openapi and re-use it here.

Reply:
not sure what you mean